README.md

```diff
@@ -96,6 +96,41 @@ build {
 }
 ```
 
+### An example with defined option `resolve_aliases`
+
+When using AWS Systems Manager parameters instead of AMI IDs in launch templates (See the [AWS documentation](https://docs.aws.amazon.com/autoscaling/ec2/userguide/using-systems-manager-parameters.html)), it is essential to set the `resolve_aliases` option to `true`. This allows the post-processor to resolve the SSM Parameter and retrieve the actual AMI ID associated with it.
+
+If `resolve_aliases` is not set to `true`, the post-processor will not detect the actual AMI ID usage, leading to potential deletion of the AMI and its related snapshot, even if they are still in use by the launch template.
```

Comment:
Unless you want to maintain backward compatibility, I don't see any reason why you would want this behavior. What do you think?

If so, it might be better to make this behavior the default. Of course, this is a breaking change in the sense that additional API permissions are required, but if the additional request occurs only in the `resolve:ssm:` case, it might be considered a bug fix.

Reply:
Personally, I think this behavior should be the default. However, if a developer other than the administrator of this post processor creates an AMI that uses SSM parameters, an error will occur unless the `ssm:GetParameters` permission is added. This seems like a breaking change to me.

Given this, I think it would be better to add the `resolve_aliases` ​​option and maintain backward compatibility.

Reply:
Yes, that seems like a breaking change. Maybe in a future major version release eg: `2.0.0` this could be a default behaviour.